test/metabase/cmd/rotate_encryption_key_test.clj

```diff
@@ -19,26 +19,33 @@
    [metabase.util.encryption :as encryption]
    [metabase.util.encryption-test :as encryption-test]
    [metabase.util.i18n :as i18n]
+   [methodical.core :as methodical]
    [toucan.db :as db]
    [toucan.models :as models])
   (:import
    (java.nio.charset StandardCharsets)))
 
 (use-fixtures :once (fixtures/initialize :db))
 
-(defn do-with-model-type
-  [mtype in-type-fns f]
-  (let [type-fns        (var-get #'models/type-fns)
-        before-type-fns @type-fns]
-    (swap! type-fns update mtype merge in-type-fns)
-    (try
-      (f)
-      (finally
-        (reset! type-fns before-type-fns)))))
```

Comment:
We had this fancy general function that was only used for one specific thing. I simplified it a bit just to handle the one thing it was used for